### ISSUE TITLE
Game session closed automatically

### ISSUE THREAD

**Issue by sudhanshugairola** [moonlight-stream/moonlight-common-c#73]
.LimeLog: Too much pending audio data: 40 ms
I/moonlight-common-c: Recovered 1 audio data shards from block 8804
D/BufferPoolAccessor2.0: bufferpool2 0xb400007d08f41628 : 4(6914048 size) total buffers - 4(6914048 size) used buffers - 0/4 (recycle/alloc) - 19/1085 (fetch/transfer)
A/libc: moonlight-common-c/src/AudioStream.c:219: void decodeInputData(PQUEUED_AUDIO_PACKET): assertion "decryptedOpusData[0] == opusHeaderByte" failed
A/libc: Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 21615 (Thread-158), pid 21195 (limelight)
I/moonlight-common-c: Audio packet queue overflow
D/PipelineWatcher: onInputBufferReleased: frameIndex not found (1101); ignored
I/moonlight-common-c: Audio packet queue overflow
I/moonlight-common-c: Audio packet queue overflow
I/moonlight-common-c: Audio packet queue overflow

**Comment by sudhanshugairola** [comment]
This error come when resolution less then 720p i think that frame conversion issue.

**Comment by sudhanshugairola** [comment]
Please check this one also

**Comment by sudhanshugairola** [comment]
Please check this one also

**Comment by cgutman** [comment]
Are you using Sunshine as your host software?

Did you compile the code yourself? This assert is only compiled in debug builds.

**Comment by sudhanshugairola** [comment]
Yes use sunshine as host. After compiled new build i got that game session closer issue

**Comment by sudhanshugairola** [comment]
This issue only for debug build.
